File: app/src/main/java/com/BlizzardArmory/warcraft/media/Realm.java

package com.BlizzardArmory.warcraft.media;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Realm {

    @SerializedName("key")
    @Expose
    private Key_ key;
    @SerializedName("name")
    @Expose
    private String name;
    @SerializedName("id")
    @Expose
    private long id;
    @SerializedName("slug")
    @Expose
    private String slug;

    public Key_ getKey() {
        return key;
    }

    public void setKey(Key_ key) {
        this.key = key;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public long getId() {
        return id;
    }

    public void setId(long id) {
        this.id = id;
    }

    public String getSlug() {
        return slug;
    }

    public void setSlug(String slug) {
        this.slug = slug;
    }

}

File: app/src/main/java/com/BlizzardArmory/warcraft/media/Realm.kt
package com.BlizzardArmory.warcraft.media

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Realm {
    @SerializedName("key")
    @Expose
    var key: Key_? = null

    @SerializedName("name")
    @Expose
    var name: String? = null

    @SerializedName("id")
    @Expose
    var id: Long = 0

    @SerializedName("slug")
    @Expose
    var slug: String? = null

}